lineman/app/app.coffee

```diff
@@ -17,10 +17,14 @@ angular.module('loomioApp', ['ngRoute',
 angular.module('loomioApp').config ($translateProvider) ->
   $translateProvider.
     useUrlLoader('/api/v1/translations/en').
-    preferredLanguage('en')
+    preferredLanguage('pt-BR')
```

Comment:
See inbox.coffee lines 6 and 7 for how to access current user.

Also I will show you how to move the translations into the layout.. the same way that other necessary data records are inserted in app/views/layouts/angular.html.haml